.travis.mk

```diff
@@ -485,3 +485,61 @@ test_jepsen: deps_debian_jepsen
 	git config --get user.email || git config --global user.email "nobody@nowhere.com"
 	cmake . -DCMAKE_BUILD_TYPE=RelWithDebInfo -DENABLE_WERROR=ON -DWITH_JEPSEN=ON
 	make run-jepsen
+
+# LuaJIT integration testing
+
+cmake_build:
+	${ARCH} cmake . -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} ${CMAKE_EXTRA_PARAMS}
+	${ARCH} cmake --build . --parallel ${NPROC}
+
+cmake_luajit_test:
+	${ARCH} cmake --build . --parallel ${NPROC} -t LuaJIT-test
+
+test_run_tarantool_test:
+	cd test && ${ARCH} ./test-run.py --vardir ${VARDIR} --force $(TEST_RUN_EXTRA_PARAMS)
+
+tarantool_linux_deps:
+	sudo apt update -y && sudo apt -y install git build-essential cmake \
+		make ninja-build zlib1g-dev libreadline-dev libncurses5-dev \
+		libssl-dev libunwind-dev libicu-dev \
+		python3 python3-six python3-gevent python3-pip
+
+tarantool_linux_tests: tarantool_linux_deps cmake_build cmake_luajit_test test_run_tarantool_test
+
+# XXX: No arch need to be set for Linux runners.
+luajit_Linux_aarch64_test: ARCH=
+luajit_Linux_aarch64_test: NPROC=$(nproc)
```

Comment:
You should use `$$`. From the documentation: `Because dollar signs are used to start make variable references, if you really want a dollar sign in a target or prerequisite you must write two of them, ‘$$’`. See [How to Use Variables](https://www.gnu.org/software/make/manual/make.html#Using-Variables).

Reply:
Fixed.